frontend/packages/kubevirt-plugin/src/selectors/dv/selectors.ts

```diff
@@ -4,20 +4,31 @@ import { V1DataVolumeTemplateSpec } from '../../types';
 import { V1alpha1DataVolume } from '../../types/api';
 import { getStorageSize } from '../selectors';
 
-export const getDataVolumeResources = (dataVolume: K8sResourceKind) =>
+// this exist to keep supporting pvc sections to existing users
+export const getDataVolumeAccessModesLegacy = (dataVolume: K8sResourceKind) =>
```

Comment:
like above it's not legacy it's PVC
```suggestion
export const getDataVolumePVCAccessModes = (dataVolume: K8sResourceKind) =>
```